frontend/jammy/handle_container.go

```diff
@@ -0,0 +1,189 @@
+package jammy
+
+import (
+	"context"
+	"encoding/json"
+	"io/fs"
+	"strings"
+
+	"github.com/Azure/dalec"
+	"github.com/Azure/dalec/frontend"
+	"github.com/Azure/dalec/frontend/pkg/bkfs"
+	"github.com/moby/buildkit/client/llb"
+	"github.com/moby/buildkit/client/llb/sourceresolver"
+	gwclient "github.com/moby/buildkit/frontend/gateway/client"
+	ocispecs "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+)
+
+const (
+	jammyRef = "mcr.microsoft.com/mirror/docker/library/ubuntu:jammy"
+
+	testRepoPath           = "/opt/testrepo"
+	testRepoSourceListPath = "/etc/apt/sources.list.d/test-dalec-local-repo.list"
+)
+
+func handleContainer(ctx context.Context, client gwclient.Client) (*gwclient.Result, error) {
+	return frontend.BuildWithPlatform(ctx, client, func(ctx context.Context, client gwclient.Client, platform *ocispecs.Platform, spec *dalec.Spec, targetKey string) (gwclient.Reference, *dalec.DockerImageSpec, error) {
+		sOpt, err := frontend.SourceOptFromClient(ctx, client)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		opt := dalec.ProgressGroup("Building Jammy container: " + spec.Name)
+
+		deb, err := buildDeb(ctx, client, spec, sOpt, targetKey, opt)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		worker, err := workerBase(sOpt, opt)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		var includeTestRepo bool
+
+		workerFS, err := bkfs.FromState(ctx, &worker, client)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		// Check if there there is a test repo in the worker image.
+		// We'll mount that into the target container while installing packages.
+		_, repoErr := fs.Stat(workerFS, testRepoPath[1:])
+		_, listErr := fs.Stat(workerFS, testRepoSourceListPath[1:])
+		if listErr == nil && repoErr == nil {
```

Comment:
Can this manual test repo handling go away once we have support for custom repos in the spec?

Reply:
Yeah this is janky AF. 😢 
I was thinking we could just copy all extra repos, but this isn't necessarily always desirable.
We could override the base image but then we need to break out the test to be something jammy specific which isn't great either.

Reply:
Hmm, I see your point, and I suppose there isn't another great solution for now. Well, maybe we can at least open an issue to revisit this? I'd definitely prefer that this not be the permanent solution